browser/guest_window_search_engine_provider_controller.h

```diff
@@ -7,6 +7,11 @@
 
 #include "brave/browser/search_engine_provider_controller_base.h"
 
+// This controller is only used by non Qwant region.
+// This class manage's newtab's ddg toggle button state.
+// Toggle button state should be reflected setting value.
+// Ex, when user changeds from ddg to others, toggle button shouldbe off and
```

Comment:
s/changeds/changes/
s/shouldbe/should be/

Reply:
Done. thanks for review!